test/parallel/test-rmdirSync-busy-loop-windows.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+require('../common');
+
+// This test ensures that rmdirSync throws EACCES instead of
+// going into a busy-loop for this scenario:
+// https://github.com/nodejs/node/issues/34580
+
+const assert = require('assert');
+const fs = require('fs');
+const { tmpdir } = require('os');
```

Comment:
The test should use [`common/tmpdir`](https://github.com/nodejs/node/tree/master/test/common#tmpdir-module) -- see other tests that read/write temporary files/directories.

Reply:
Thank you. Done. :)